CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+
+
+## General workflow for contributing ##
+
+1. Fork, then clone the repo:
+```
+git clone git@github.com:your-username/exiv2.git
+```
+
+2. Configure the project and build (More details about how to run CMake at XXX) :
+```
+mkdir build && cd build
+cmake .. && cmake --build . --config Release
+```
+
+3. Make sure the tests pass:
+```
+make tests			# Application tests
+./bin/unit_tests	# Unit tests
+```
+
+4. Make your change. Add tests for your change. Make the tests pass.
+5. Create the Pull Request (PR).
+	a. Suggest a reviewer if you know that some of the mantainers know about the specific change.
+    b. Check if the CI checks pass.
+    c. If your PR lives for a long time, try to avoid to press the button "Update branch" in the Pull Request view. We prefer rebasing in top of master, instead of merging master into branches.
+
```

Comment:
Indents for `a.` do not match. Reviewers can only be suggested by project members, right?

Reply:
Good point. I think you are right. I will remove that point and just mention that a project member will review the PR ASAP.